src/main/webapp/i18n/de/editor.json

```diff
@@ -42,6 +42,7 @@
             "loadingFiles": "Lade Dateien",
             "fileLoadedSuccess": "Datei erfolgreich geladen",
             "fetchingParticipation": "Deine Teilnahme wird geladen, bitte warte einen Moment...",
+            "tabSize": "Tabulatorweite",
```

Comment:
You're either missing a colon here or you accidentally inserted one in the englisch version

Reply:
Good catch! Yes, there is supposed to be one here.